src/solver/infeasible-problem-analysis/unfeasible-pb-analyzer.cpp

```diff
@@ -0,0 +1,49 @@
+#include <fstream>
+#include <algorithm>
+
+#include "unfeasible-pb-analyzer.h"
+#include "variables-bounds-consistency.h"
+#include "constraint-slack-analysis.h"
+#include <antares/logs/logs.h>
+
+
+using namespace operations_research;
+
+namespace Antares::Optimization
+{
+
+UnfeasiblePbAnalyzer::UnfeasiblePbAnalyzer(std::vector<std::shared_ptr<UnfeasibilityAnalysis>> analysisList)
+        : analysisList_(analysisList)
```

Comment:
is there a reason to pass ``analysisList by value?

Reply:
**analysisList** is a **vector** of unique  pointers, copying this vector is not copying the objects it contains but only the pointers. And to be accurate, they are not **copied**, but moved (see call of **std::move(...)**)